api/src/main/proto/bio/terra/tanagra/underlay/mapping.proto

```diff
@@ -0,0 +1,74 @@
+// Messages mapping logical entities to physical tables
+
+syntax = "proto3";
+
+package bio.terra.tanagra.underlay;
+option java_package = "bio.terra.tanagra.proto.underlay";
+option java_multiple_files = true;
+
+import "bio/terra/tanagra/underlay/entities.proto";
+import "bio/terra/tanagra/underlay/tables.proto";
+
+// An id to uniquely idnetify a column of a table within an underlay.
+message ColumnId {
+  // The name of the dataset.
+  string dataset = 1;
```

Comment:
opt: A more portable term for this is probably `schema`. I don't really love this term though, so I'm fine leaving dataset, even if it might be slightly overfitting to BigQuery terminology (AFAIK)

Reply:
It matches for Postgres & MySQL as well, so sticking with it for now.